starters/apps/basic/src/components/header/header.tsx

```diff
@@ -8,23 +8,23 @@ export default component$(() => {
   return (
     <header>
       <div class="logo">
-        <a href="https://qwik.builder.io/" target="_blank">
+        <a rel='noopener' aria-label='logo' href="https://qwik.builder.io/" target="_blank">
```

Comment:
```suggestion
        <a href="https://qwik.builder.io/" target="_blank" rel="noopener">
```

`href` is the most important attribute in `<a>`, so let's give it the first place. Also use double qoutes.

when it comes to `aria-label`, I think the `"logo"` itself doesn't mean anything. I think the link can have `"Qwik homepage"` aria-label.

svg itself can have `<title>Qwik logo</title>` inside and `role="img"` but it's irrelevant.